src/sidebar/components/HelpPanel.tsx

```diff
@@ -78,11 +80,10 @@ function HelpPanel({ auth, session }) {
     tutorial: 'Getting started',
     versionInfo: 'About this version',
   };
+  type PanelKey = keyof typeof subPanelTitles;
```

Comment:
I'm not sure exactly why using `keyof subPanelTitles` works with JS but not TS, but it seems useful to give the type a name in any case.